swagger.yaml

```diff
@@ -455,6 +455,10 @@ paths:
           description: Error.
           schema:
             $ref: '#/definitions/error'
+        "422":
```

Comment:
Not sure I understand this change. When would we use this?

Reply:
Yeah, subsystem tests are kinda annoying to run locally, but you should be able to just get the system up and running in minikube with `make deploy-test` then just run the query the subsystem test is running manually.

Reply:
Also, however you want to handle testing is fine, I just wasn't sure if I should review this or if you were just testing, maybe mark the whole PR as WIP until you want reviews on the changes?